jwst/transforms/models.py

```diff
@@ -36,12 +36,12 @@
 Slit = namedtuple('Slit', ["name", "shutter_id", "dither_position", "xcen", "ycen",
                            "ymin", "ymax", "quadrant", "source_id", "shutter_state",
                            "source_name", "source_alias", "stellarity",
-                           "source_xpos", "source_ypos", ])
+                           "source_xpos", "source_ypos", "source_ra", "source_dec"])
 """ Nirspec Slit structure definition"""
 
 
 Slit.__new__.__defaults__ = ("", 0, 0, 0.0, 0.0, 0.0, 0.0, 0, 0, "", "", "", "",
```

Comment:
It looks to me as if there should only be three empty strings at the end of this line, i.e. for shutter_state, source_name, and source_alias.  The definition of `Slit` has 17 elements, but this specification of default values has 18 elements.
There are some doc strings in this file that could be updated.

Reply:
fixed

Reply:
Which docstrings did you have in mind?

Reply:
In transforms/models.py, both `Gwa2Slit` and `Slit2Msa` have a `slits` parameter, and the "Parameters" section of their doc strings gives the contents of those tuples.

Reply:
Thanks, I didn't notice them. All comments have been addressed now.